DVR/SessionDataTask.swift

```diff
@@ -35,7 +35,12 @@ class SessionDataTask: NSURLSessionDataTask {
         // Find interaction
         if let interaction = cassette?.interactionForRequest(request) {
             // Forward completion
-            completion?(interaction.responseData, interaction.response, nil)
+            if let completion = completion {
+                let queue = dispatch_queue_create("com.venmo.DVR.sessionDataTaskQueue", nil)
```

Comment:
Not sure which aspect of the queue you mean when you say non-deterministic. Either way, a session manages just one queue for its lifetime so I think mirroring that within our own `Session` object would make sense.